hack/builder/rsyncd.conf

```diff
@@ -0,0 +1,19 @@
+gid = 0
+uid = 0
+log file = /dev/stdout
+reverse lookup = no
+[build]
+    hosts allow = *
+    read only = false
+    path = /root/go/src/kubevirt/hyperconverged-cluster-operator
```

Comment:
What do we have in this path? Nothing is there from the Dockerfile. Also,the conversion of ${HOME}/go/src/project-name is very old and we don’t have to use it.